federation/pkg/crud/util/crudhelper.go

```diff
@@ -0,0 +1,215 @@
+/*
```

Comment:
No, I'm not. I'm seeing a new file rather than a diff from the old file. Lots of green, and St. Patrick's Day is over now.